common/types/mapper/thrift/errors.go

```diff
@@ -67,6 +67,8 @@ func FromError(err error) error {
 		return FromRetryTaskV2Error(e)
 	case *types.ServiceBusyError:
 		return FromServiceBusyError(e)
+	case *types.UserServiceBusyError:
```

Comment:
this file is missing unit tests. since you are touching it would be great to add some basic tests. maybe in a follow up PR. CDNC-8058

Reply:
I have taken this task